### PR TITLE
MAGECLOUD-4486: Docker functional tests and CI with ece-tools test framework

### PR DESCRIPTION
**Pull request by BaDos**
### Fixed Issues (if relevant)

https://magento2.atlassian.net/browse/MAGECLOUD-4974

### Contribution checklist
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds on Travis CI are green)

### CODE REVIEW

File: src/Config/Schema.php [hunk 1]
```diff
@@ -89,8 +89,8 @@ public function getDefaults(string $stage): array
      */
     public function getSchema(): array
     {
-        return $this->parser->parseFile(
-            $this->systemList->getConfig() . '/schema.yaml',
+        return $this->parser->parse(
+            file_get_contents($this->systemList->getConfig() . '/schema.yaml'),
```
Comment (shiftedreality) [inline src/Config/Schema.php]: Please use File.php

**Review by shiftedreality — CHANGES_REQUESTED** [review]

File: src/Config/Schema.php (hunk 1 above)
Comment (BaDos) [reply src/Config/Schema.php]: Done

**Review by shiftedreality — APPROVED** [review]

**Review by oshmyheliuk — APPROVED** [review]

**Comment by shiftedreality** [comment]
No QA needed

**Comment by ghost** [comment]
Hi @BaDos, thank you for your contribution! </br>Please, complete [Contribution Survey](https://opensource.magento.com/survey/contribution/magento+ece-tools+656), it will take less than a minute. </br>Your feedback will help us to improve contribution process.

**Pull request merged**